tutorials/5 All QRenderers/5.1 Ansys/Ansys QRenderer.ipynb

```diff
@@ -91,25 +111,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Allow running the same cell here multiple times to overwrite changes\n",
+    "# Enable overwrite, so that we can re-un this cell multiple times without incurring object name collisions\n",
     "design.overwrite_enabled = True\n",
     "\n",
-    "## Custom options for all the transmons\n",
-    "options = dict(\n",
-    "    # Some options we want to modify from the defaults\n",
-    "    # (see below for defaults)\n",
+    "# We now define the same custom options for all the transmons.\n",
+    "# We only list the options we intend to modify with respect to the default values\n",
+    "options = Dict(\n",
     "    pad_width = '425 um', \n",
     "    pocket_height = '650um',\n",
-    "    # Adding 4 connectors (see below for defaults)\n",
-    "    connection_pads=dict(\n",
+    "    # Adding 3 connectors\n",
```

Comment:
Instead of `# Adding 3 connectors`

Suggest:
`# Adding 3 connection_pads for each qubit.`